app/cronjob/trailingTrade/step/handle-open-orders.js

```diff
@@ -158,17 +123,6 @@ const execute = async (logger, rawData) => {
             moment().toISOString()
           );
         }
-      } else if (await isExceedingMaxOpenTrades(logger, data)) {
```

Comment:
@habibalkhabbaz This part is necessary to prevent new buy orders to be placed. It took me almost 30 minutes to recognize why I'm getting buy orders over the limit.